spec/factories/customers.rb

```diff
@@ -1,7 +1,7 @@
-FactoryBot.define do 
+FactoryBot.define do
   factory :customer do
-    first_name { Faker::JapaneseMedia::DragonBall.character }
-    last_name { Faker::JapaneseMedia::DragonBall.character }
+    first_name { Faker::Name.first_name  }
```

Comment:
interesting that the other group was utilizing a .unique here with the faker data to ensure that they didn't have duplicate names